tests/test_suite.py

```diff
@@ -109,6 +108,12 @@ def parse_args():
                                    "ask for subset of all rules this way. (If you "
                                    "type ipv6 as a target, all rules containing "
                                    "ipv6 within id will be performed."))
+    parser_rule.add_argument("--manualdebug",
+                             dest="manual_debug",
+                             action="store_true",
+                             help=("Stop execution when error is encountered, so "
+                                   "developer can connect to VM / container with "
+                                   "scenario set up"))
```

Comment:
you use 2 different quote styles in 2 hunks in a single patch

Reply:
alternative help:
```
If an error is encountered, all execution on the VM / container will pause to allow debugging.
```

Reply:
Good notes.